### DISCUSSION TITLE
Do we support Microsoft.Azure.DurableTask.AzureStorage version in .net 6.0?

### DISCUSSION THREAD

**Discussion by yogirajdeshpande** [Azure/azure-functions-durable-extension#2089] [Q&A] [answered]
Microsoft.Azure.DurableTask.AzureStorage having dependency on window.azure .storage which is deprecated. So what is the alternative for that?

**Comment by cgillum** [comment, marked as answer]
I think there are two independent questions here, so I'll try to answer them both:

> Do we support Microsoft.Azure.DurableTask.AzureStorage version in .net 6.0?

Yes, absolutely. These libraries target .NET Standard 2.0 and are fully compatible with .NET 6.0. In fact, many users are actively running Durable Functions with this Azure Storage dependency in the Azure Functions 4.0 host, which is .NET 6.

> Microsoft.Azure.DurableTask.AzureStorage having dependency on window.azure .storage which is deprecated. So what is the alternative for that?

There is no alternative right now. Moving to the newer storage SDKs is on the backlog, but it's not a high priority yet since using the older, deprecated libraries doesn't cause any particularly bad problems for us.
